code/modules/clothing/glasses/hudglasses.dm

```diff
@@ -24,7 +24,7 @@
 
 /obj/item/clothing/glasses/hud/dropped(mob/living/carbon/human/user)
 	..()
-	if(user.glasses != src)
+	if(istype(user) && user.glasses != src)
 		return
```

Comment:
when is this not true?

Reply:
when new_players late spawn with cheap sunglasses :)